server/s3_aws_etag.cc

```diff
@@ -128,3 +128,8 @@ std::string S3AwsEtag::finalize() {
 }
 
 std::string S3AwsEtag::get_final_etag() { return final_etag; }
+
+int S3AwsEtag::get_num_of_parts(std::string etag)
+{
+    return std::stoull(etag.substr(etag.rfind('-') + 1));
```

Comment:
Looks like you moved this logic into a helper function.  Excellent.  Thank you.  Can you please also add a comment specifying what the expected string format is to help the reader understand the convention?

Reply:
Added documentation just before `S3AwsEtag` class.

Reply:
Thanks.  Resolving conversation.